app/models/shelter_review.rb

```diff
@@ -0,0 +1,6 @@
+class ShelterReview < ApplicationRecord
+  belongs_to :shelter
+  validates_presence_of :title
```

Comment:
Lines 3-5 can be combined, like one would do with an attr_reader. This is just a minor suggestion, and not at all priority. I thought I would mention it, in case you see it somewhere else.

Reply:
Nice! I did not know that!